src/Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN rm requirements.txt
 # Wheelhouse cleanup.
 RUN rm -rf /opt/wheelhouse
 
+# install newer sqlalchemy than gobcore
+RUN pip install SQLAlchemy==2.0.18
```

Comment:
Ik neem aan dat de SQLAlchemy update via Dockerfile tijdelijk is, tot SQLAlchemy in Core naar 2.0 ge-update kan worden?
Is er nog bezwaar tegen om hier `SQLAlchemy~=2.0.18` te maken?

Reply:
klopt, zo beperk ik dit tot upload. Anders zal je bij de volgende core update ook bijvoorbeeld api moeten omzetten (en core zelf in 1e instantie)
zal er `~=` van maken